BenchPress/Generators/LanguageProviders/PowershellLanguageProvider.cs

```diff
@@ -49,7 +54,7 @@ public string SDK(SDKFunction sdkFunction)
         switch (sdkFunction.Kind)
         {
             case TestType.ResourceExists:
-                return $"Get-{sdkFunction.ResourceType.FunctionPrefix}Exists";
+                return $"Confirm-AzBPResource";
             case TestType.Location:
                 return $"Check-{sdkFunction.ResourceType.FunctionPrefix}Location";
```

Comment:
Yea I was debating on removing them or not. I have adding Custom Assertion generation as an Issue I will create, so I will remove references of this Location test generation now.

Reply:
Sounds good. I will offer that the file that made me do a double take on this was `TestGenerator.cs` and the `GenerateCheckLocationViewModel` method. Any thoughts on thatg?

Reply:
Yup! Will be removing any functionality that generates these "Location" type tests since these generated tests wouldn't work anymore and makes all the underlying logic no longer functional.

Reply:
Removed all functionality regarding generating a Location Test type.